domains/hongsondev.json

```diff
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "4h3w",
+           "discord": "1221370693959286892",
+           "OWL": "eyJlbmMiOiJBMTI4Q0JDLUhTMjU2IiwiYWxnIjoiUlNBLU9BRVAiLCJraWQiOiJaa1VsRmRqVThiUEstLXVVM2JJR09PVHFYYVFFS1ZINFVXOW53MTR6WTJnIn0.dJXIG4-bVmaJ2CB9-b242WybBrXrdP1C3Mey6St57zKwbCEn3XI90TzOQOYsMVXC1FIL2s7AgMkWxqZOsRnlKCrBk3v0cnj_ozAgOPFRO2G02p4DLSFoukkCsQLEdorWwwuDuaXjzVsLL_MBcsxuvIqDozE3hRi1PzK4BHdUETifnjYgbs7RmkifEqURP0vyHRnfwkhXfJdSwPGu1gNKXojiahQjmIQImfO48w5O_CO89QrFG1nQizcUthWg-gj_rtP0yVhD1hW6HPxlKj1UgzLCtbNKG0hqVd5zK1zLMfZwG0t0Wzher-TCgOvPfCqTc3UhrJ7k7iEM5MvLtodeCQ.7tx8nX3uiE-PxbwNtZ5LnQ.-sdjI1Wlf_yGWMtNfwd3CWx95ibPOmjZz8EM87CLCGTbx8SQkeh-A5fXdCcRA8whFJ9kG_YjM3MLcr5KgfW7QA.XV1IaRdBkVNDtM9kLo4aYA"
+        },
+    
+        "record": {
+            "A": ["185.27.134.143"]
```

Comment:
I cannot reach your website, 185.27.134.143 didn’t send any data.

Reply:
> I cannot reach your website, 185.27.134.143 didn’t send any data.

Its an iFastNet website (a.k.a. InfinityFree website, it will only work through the domain.

Reply:
Then user would need to provide a website preview.

Reply:
> Then user would need to provide a website preview.

I will try to use a custom record on my local network DNS and provide a website preview.

Reply:
Website tries to redirect to HTTPS but there is no valid SSL certificate which ends up in an error @CuteDog5695 

![image](https://github.com/user-attachments/assets/268863d8-0e20-48b1-be34-875caac77cdb)